src/metabase/models/card.clj

```diff
@@ -135,6 +137,38 @@
    :id
    {:default 0}))
 
+(methodical/defmethod t2/batched-hydrate [:model/Card :in_dashboards]
+  [_model k cards]
+  (mi/instances-with-hydrated-data
+   cards k
+   #(->> (t2/query {:select [:card_id
```

Comment:
it's hard to scan for a % somewhere inside of this, could we promote this into a `(fn []`?